lightningd/channel_control.c

```diff
@@ -2247,6 +2439,24 @@ static const struct json_command splice_signed_command = {
 };
 AUTODATA(json_command, &splice_signed_command);
 
+static const struct json_command stfu_channels_command = {
+	"stfu_channels",
+	"channels",
+	json_stfu_channels,
+	"Put {channel_ids} channels into stfu mode and return their available"
+	" balance."
+};
+AUTODATA(json_command, &stfu_channels_command);
```

Comment:
These are *definitely* dev commands!!

Reply:
And since you only ever call it with one channel, `dev_quiesce` would work.

Reply:
🤔 Discussing the trade offs of exposing stfu / abort in this comment over here: https://github.com/ElementsProject/lightning/pull/7083#issuecomment-2467091184